lib/aquatone/collectors/censys.rb

```diff
@@ -0,0 +1,83 @@
+module Aquatone
+  module Collectors
+    class Censys < Aquatone::Collector
+      self.meta = {
+        :name         => "Censys",
+        :author       => "James McLean (@vortexau)",
+        :description  => "Uses the Censys API to find hostnames in TLS certificates",
+        :require_keys => ["censys","censysid"],
```

Comment:
I'm sorry, just one more request: Could we change the key names to `censys_id` and `censys_secret` to be consistent with Censys' own wording and other aquatone collectors?

Thanks!